### PR TITLE
feat(metrics/family): add `get` method to Family

### PR DESCRIPTION
**Pull request by ethercflow**
Address comments in https://github.com/prometheus/client_rust/pull/230#issuecomment-2440804265

### CODE REVIEW

**Comment by ethercflow** [comment]
@mxinden PTAL, thanks!

**Comment by cratelyn** [comment]
seconding this addition, i would also suppport this change, and would like have the ability to avoid creating new metrics in a family when looking up the series for a particular label set. i'll cross-reference #244, which also introduces a new accessor to `Family<S, M, C>`.

File: src/metrics/family.rs [hunk 1]
```diff
@@ -452,4 +469,47 @@ mod tests {
                 .get()
         );
     }
+
+    #[test]
+    fn test_get() {
+        let family = Family::<Vec<(String, String)>, Counter>::default();
+
+        // Test getting a non-existent metric
```
Comment (mxinden) [inline src/metrics/family.rs:477]:
```suggestion
        // Test getting a non-existent metric.
```

Nit, convention in the codebase, please also apply below.

**Review by mxinden — COMMENTED** [review]
Thank you for the thorough patch.

What do you think of replacing the code below in `Family::get_or_create` with a call to `Family::get`?

```
        if let Ok(metric) =
            RwLockReadGuard::try_map(self.metrics.read(), |metrics| metrics.get(label_set))
        {
            return metric;
        }
```

Also, please add a changelog entry.

**Comment by ethercflow** [comment]
@mxinden Addressed, PTAL, thanks!

**Review by mxinden — APPROVED** [review]
Thank you!

**Pull request merged**